src/models/account.ts

```diff
@@ -36,6 +36,17 @@ class Account {
     return _accountObject
   }
 
+  public static async getTotalNumber(): Promise<number> {
+    // This matches 'account:' followed by a UUID, with nothing after it.
+    // Redis's pattern matching is fairly limited; the best way I found to do this is
+    // to individually match each character of the last part (12 alphanumeric characters) of the UUID
+    const pattern = `account:*-*-*-*-[a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9][a-z0-9]`
```

Comment:
Nitpick but this should be `_pattern`

Also, could we not use `account:*` as the pattern to match against?

Reply:
No; that would match `account:${uuid}` keys as desired, but it would also match `account:${uuid}::block:${blockName}` keys, so blocks within accounts would be included in the count.